OG_Format.adoc

```diff
@@ -440,7 +439,7 @@ If the glider has no <platform_name> use <platform_serial_number> instead to cre
 * long_name = “unique identifying name of glider”;
  |mandatory
 
- The  platform name should be constructed from the manufacturer prefix and the platform serial number e.g. "sea001" (seaglider), unit001 (slocum), sea001 (seaexplorer), sp001 (spray). Where the serial number of the platform is not known, a local nickname can be used e.g. "orca", "sverdrup", "ammonite".
+ The  platform name should be constructed from the manufacturer prefix and the platform serial number e.g. "sea001" (seaglider), unit_001 (slocum), sea001 (seaexplorer), sp001 (spray). Where the serial number of the platform is not known, a local nickname can be used e.g. "orca", "sverdrup", "ammonite".
```

Comment:
can you update the seaglider example  - e.g. "sg638" (seaglider)

Reply:
resolved